elephant/conversion.py

```diff
@@ -152,3 +155,704 @@ def binarize(spiketrain, sampling_rate=None, t_start=None, t_stop=None,
     else:
         return res, pq.Quantity(np.arange(t_start, t_stop+sampling_period,
                                           sampling_period), units=units)
+
+###############################################################################
+#
+# Methods to calculate parameters, t_start, t_stop, bin size, number of bins
+#
+###############################################################################
+def calc_tstart(num_bins, binsize, t_stop):
```

Comment:
When I mean private, I mean with a single underscore.  So instead of `calc_tstart`, it would be `_calc_tstart`.  This hides it by default, but anyone and everyone who wants to use it can.  But it could be changed or removed at any time.